src/convertToObject.js

```diff
@@ -1,18 +1,17 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  sourceString.split(';').forEach(index => {
```

Comment:
index - it's a second params
![image](https://user-images.githubusercontent.com/91411692/225931531-16834d20-a823-4b5c-bf57-ea85103a9c6f.png)

```suggestion
  sourceString.split(';').forEach(... => {
```